packages/snowpack/src/commands/build.ts

```diff
@@ -147,55 +146,56 @@ export async function command(commandOptions: CommandOptions) {
 
   for (const [dirDisk, dirDest, allFiles] of includeFileSets) {
     for (const locOnDisk of allFiles) {
-      const {baseExt: fileExt} = getExt(locOnDisk);
-      let outLoc = locOnDisk.replace(dirDisk, dirDest);
-      let builtLocOnDisk = locOnDisk;
-      const extToReplace = config._extensionMap[fileExt] || srcFileExtensionMapping[fileExt];
-      if (extToReplace) {
-        outLoc = replaceExt(outLoc, extToReplace);
-        builtLocOnDisk = replaceExt(builtLocOnDisk, extToReplace);
-      }
+      const replaceSrcExt = new RegExp(path.extname(locOnDisk) + '$');
       const builtFileOutput = await buildFile(locOnDisk, {
         plugins: config.plugins,
         messageBus,
         isDev: false,
         isHmrEnabled: false,
       });
       allBuiltFromFiles.add(locOnDisk);
-      const {baseExt, expandedExt} = getExt(outLoc);
-      let contents =
-        builtFileOutput[
-          config._extensionMap[fileExt] || srcFileExtensionMapping[fileExt] || fileExt
-        ];
-      if (!contents) {
-        continue;
-      }
-      const cssOutPath = outLoc.replace(/.js$/, '.css');
-      mkdirp.sync(path.dirname(outLoc));
-      switch (baseExt) {
-        case '.js': {
-          if (builtFileOutput['.css']) {
-            await fs.mkdir(path.dirname(cssOutPath), {recursive: true});
-            await fs.writeFile(cssOutPath, builtFileOutput['.css'], 'utf-8');
```

Comment:
See above comment: we now handle `.css` in a separate pass, so we don’t need it. I think the end result is cleaner / easier to follow